ocs_ci/deployment/deployment.py

```diff
@@ -734,6 +735,9 @@ def post_ocp_deploy(self):
         )
         if ibmcloud_ipi:
             ibmcloud.label_nodes_region()
+        # configure Ingress Node Firewall and restrict SSH access to nodes
+        if config.ENV_DATA.get("restrict_ssh_access_to_nodes", False):
```

Comment:
Add condition for OCP >= 4.16 as for upgrade we install 4.15 from release-4.16 branch and it's only supported from 4.16.